cosmos/operators/azure_container_instance.py

```diff
@@ -66,7 +66,7 @@ def __init__(
     def build_and_run_cmd(self, context: Context, cmd_flags: list[str] | None = None) -> None:
         self.build_command(context, cmd_flags)
         self.log.info(f"Running command: {self.command}")
-        result = super().execute(context)
+        result = AzureContainerInstancesOperator.execute(self, context)
```

Comment:
@danielvdende is there any chance you could get some test coverage over here?

Reply:
I'm not sure what the scope of the test would be tbh. I could add tests for this function, but it would essentially be validating that a number of mocks were being called. Is that what you have in mind?

Reply:
@tatiana gentle reminder about this :-). Would be great if we can merge this PR soon

Reply:
Hi @danielvdende it would be great if you could add a test that reproduces the error you were facing before making this fix:
```
The inheritance order of the DbtAzureContainerInstanceBaseOperator was incorrect, as well as the wrong execute method being called.
```
Ideally, this test would fail before your fix and pass after your fix. Unless we have a test, we're at risk of incurring the same error again in the future.

Reply:
Hi @tatiana , I've added a test that covers this. It indeed fails if I run it on code as it was before this fix.